packages/agents/src/tests/alarms.test.ts

```diff
@@ -0,0 +1,26 @@
+import { env } from "cloudflare:test";
+import { describe, expect, it } from "vitest";
+import type { Env } from "./worker";
+
+declare module "cloudflare:test" {
+  interface ProvidedEnv extends Env {}
+}
+
+describe("scheduled destroys", () => {
+  it("should not throw when a scheduled callback nukes storage", async () => {
+    const agentId = env.TestDestroyScheduleAgent.idFromName(
+      "alarm-destroy-repro"
+    );
+    let agentStub = env.TestDestroyScheduleAgent.get(agentId);
```

Comment:
use getAgentByName here please? (in general we can never use a raw stab since it skips initialisation)